src/js/wall/FirstUser.jsx

```diff
@@ -12,11 +11,23 @@ const initialValues = {
     password: "",
 };
 
-export const FirstUser = ({ onSubmit, errors }) => {
+/**
+ * Responsible for creating the first Virtool user
```

Comment:
Think of component docstrings like you are telling another developer what they will get from it.
```suggestion
 * A form for creating the first instance user.
```
Take a look at these docs for examples like:
* A vertically stacked set of interactive headings that each reveal a section of content.
* A date field component that allows users to enter and edit date.
* A control that allows the user to toggle between checked and not checked.